lib/qmp.mli

```diff
@@ -35,6 +35,24 @@ type fd_info = {
   fdset_id : int;
 }
 
+(** QOM properties - https://elmarco.fedorapeople.org/qemu-qmp-ref.pdf*)
```

Comment:
Please use the official documentation url. Either the root or the specific linked page:
- https://wiki.qemu.org/index.php/Documentation
- https://qemu.weilnetz.de/doc/qemu-qmp-ref.html

Reply:
updated